app/controllers/api/v1/recipes_controller.rb

```diff
@@ -13,4 +13,18 @@ def index
     end
       render json: @recipes, :include => [:user]
   end
+
+  def create
```

Comment:
Since only users can create recipes, we'll want to reflect that in the API by only allowing authenticated requests to this endpoint. Check out https://github.com/plataformatec/devise/wiki/How-To:-Define-resource-actions-that-require-authentication-using-routes.rb

Essentially, we can either require authentication at the route level or at the controller level. I think controller-level might be more succinct, but either way is fine.